### ISSUE TITLE
URI probably should be encoded

### ISSUE THREAD

**Issue by eduardb** [deanoemcke/thegreatsuspender#569]
Hi! An example of a suspended tab has the following URL: chrome-extension://klbibkeccnjlkjkiokjodocebajanakg/suspended.html#ttl=Extensions%20-%20Kotlin%20Programming%20Language&uri=https://kotlinlang.org/docs/reference/extensions.html#nullable-receiver

Notice how the URL has two `#` signs in it, making it invalid, technically speaking. I experienced a problem with this while using another extension, The Great Suspender, in which I couldn't import such URLs.

I think that to avoid this problem, the `uri` should be URL encoded, just like the `ttl` is.

**Comment by deanoemcke** [comment]
Yeah. The uri is purposely not encoded. It's to provide a (very lo-fi) way to access a suspended tab on a synced device.

Because this extension does not support chrome history sync (https://github.com/deanoemcke/thegreatsuspender/issues/46) I did the uri this way so that at the very least you could manually delete everything before the &uri= to retrieve the original website.

It's pretty hack I know. But not sure if the session buddy import issue is really worth worrying about?

**Comment by eduardb** [comment]
Thanks for your reply. It makes sense now. 
That's what I ended up doing, I did a `Replace all` to strip out everything before `&uri=`. But I can imagine that some users will give up before that. 
Please feel free to close the issue if you feel like it. Cheers!

**Comment by deanoemcke** [comment]
I'll leave this open, as I will encode the url again once I manage to resolve iss #46 (sync chrome history across devices)
